src/js/utils/clock.ts

```diff
@@ -1,6 +1,6 @@
 import { now as getTime } from 'utils/date';
 
-const performance = window.performance || {
+const performance: Performance | any = window.performance || {
```

Comment:
Semantically the same as `performance: any`.

Reply:
I can change to `Performance | GenericObject`.

Reply:
done